Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2633 -- Remove Storybook

Resolves [EREGCSC-2633](https://jiraent.cms.gov/browse/EREGCSC-2633)

**Description**

Storybook hasn't been used or updated in quite some time; we also aren't likely to use it or update it anytime in the near future.  As such, we're simply removing it.

If we miss it, we can always re-implement the latest version from scratch (or any of the Storybook alternatives that have popped up over the past few years).

**This pull request changes:**

- Removes artifacts and files related to our Storybook implementation

**Steps to manually verify this change:**

1. Green check marks
2. Storybook is no longer found in the codebase
3. Storybook can no longer be run locally using the `make storybook` CLI command
